### DISCUSSION TITLE
Chemical kinetics

### DISCUSSION THREAD

**Discussion by JMCarmona10** [reaktoro/reaktoro#278] [Q&A]
I have two questions regarding defining reactions in the section of chemical kinetics: 
1. Once i have define the chemical system with one specific reaction (reaction rate). In that same environment is it possible to add another reactions with its own parameters while keeping the previous reactions into consideration? 
2. Lets say i have to create a new program for each reactions is it possible to combine all the reactions into one. For example, i have calcite as the rock but i would like to show all possible reactions with different specie.

**Comment by allanleal** [comment]
Hi @JMCarmona10 , I'm not sure if I understood you correctly. It's always best to show some snippets of codes (preferably complete and minimal, but I understand that in your case right now a complete working code does not make sense).

> Once i have define the chemical system with one specific reaction (reaction rate). In that same environment is it possible to add another reactions with its own parameters while keeping the previous reactions into consideration?

Yes, it is possible to define multiple reactions within the same chemical system, each with its own set of parameters.

> Lets say i have to create a new program for each reactions is it possible to combine all the reactions into one. For example, i have calcite as the rock but i would like to show all possible reactions with different specie.

I did not understand this. Do you want to create a lumped reaction (say, a single reaction of mineral dissolution containing different rate mechanisms that otherwise would require more than one reaction to be defined, one for each mechanism?).
